docs/en/setup/Run-E2E-Tests.md

```diff
@@ -31,9 +31,23 @@ e2e cleanup
 
 To use skywalking-infra-e2e in GitHub Actions, add a step in your GitHub workflow.
 
+The working directory is uploaded to GitHub Action Artifact after the task is completed, which contains environment variables and container logs in the environment.
+
 ```yaml
 - name: Run E2E Test
   uses: apache/skywalking-infra-e2e@main      # always prefer to use a revision instead of `main`.
   with:
-    e2e-file: e2e.yaml                        # need to run E2E file path
+    e2e-file: e2e.yaml                        # (required)need to run E2E file path
+    log-dir: /path/to/log/dir                 # (not required)the container logs path, if not provide it would be auto generation
```

Comment:
```suggestion
    log-dir: /path/to/log/dir                 # (not required)the container logs path, if not provide it would be auto generation
```

What is the default path?

Reply:
If this job contains `matrix`, it would be: `<work_dir>/logs/<job_name>_<matrix_value>`
if not, it would be `<work_dir>/logs/<job_name>`.

Reply:
What is the path for running locally?

Reply:
```suggestion
    log-dir: /path/to/log/dir                 # (Optional). Use `<work_dir>/logs/<job_name>_<matrix_value>`(if have GHA matrix) or `<work_dir>/logs/<job_name>` in GHA, and output logs into `xxxx` out of GHA env, such as  running locally.
```

Please fill the `xxx` and fix the sentence. Basically for the document, we prefer the complete sentence. Grammarly tool(on Chrome) is super helpful, I thin.

Reply:
> What is the path for running locally?

It `<work_dir>/logs`, have been updated to the document.